testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/realm/RealmTest.java

```diff
@@ -195,6 +196,36 @@ public void createRealmFromJson() {
         adminClient.realms().realm("admin-test-1").remove();
     }
 
+    @Test
+    public void createRealmWithPasswordPolicyFromJsonWithInvalidPasswords() {
+        //try to create realm with password policies and users with plain-text passwords what doesn't met the policies
+        RealmRepresentation rep = loadJson(getClass().getResourceAsStream("/import/testrealm-keycloak-6146-error.json"), RealmRepresentation.class);
+        
+        boolean thrownBRException  = true;
+                try {
+            adminClient.realms().create(rep);
+            thrownBRException = false;
```

Comment:
Use `@Rule ExpectedException` instead, see e.g. https://github.com/keycloak/keycloak/blob/fd025ae76b96c0bc8798deb46b184115dd327fd0/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/group/GroupTest.java#L512

Reply:
Thank you @hmlnarik, incorporated.